### DISCUSSION TITLE
Restart files not created by Starter

### DISCUSSION THREAD

**Discussion by JAWilliams123** [OpenRadioss/OpenRadioss#915] [Q&A] [answered]
Hello,

I am very happy that Radioss is available through the OpenRadioss format...Many thanks!
I downloaded the Windows binaries but keep getting an error stating that the restart file is not found. For example, when running the tensile_test_section example I get the following error: 
 
** ERROR: FILE tensile_LAW2_0000_0001.rst NOT FOUND     ERROR TERMINATION

I have set up the paths as per INSTALL.md but have not been able to figure out what is causing the issue. Any help would be greatly appreciated.

Many thanks,
JWilliams

**Comment by MarianBulla** [comment, marked as answer]
Hello JWilliams,
Thank you very much for trying OpenRadioss.
According to the ERROR message, the engine is not able to find the restart file, which is usually generated by the starter.
So, did you run the starter with that model, first ?
If so, did it finalize without any error ?
Thanks and best regards,
Marian

**Reply by JAWilliams123** [reply]
Hello Marian,
Yes, I executed the starter first but there is no feedback to the cmd shell. I tried both the default starter_win64.exe as well as the single precision starter_win64_sp.exe but had the same outcome, i.e., no output files or shell out. The starter was called using the format given below.  
`starter_win64.exe -i starter_file_0000.rad`

Many thanks,
JWilliams

**Reply by MarianBulla** [reply]
Hell JWilliams,
did you set all the environment variables, as described in the "How to run OpenRadioss" section ?
If "yes", then you should see something like this from the starter (see image below).
Hope it helps you ?

![image](https://user-images.githubusercontent.com/105412502/208294113-3366456d-c6b0-4111-a91b-97139543f965.png)

Thanks and best regards,
Marian

**Reply by JAWilliams123** [reply]
Hello Marian,
Many thanks again. Yes, all environmental variables were set as per "How to run OpenRadioss" but had the path in my *.bat file in quotation marks (" "). After removing the quotation marks everything was up and running.

Best wishes,
JWilliams.

**Reply by MarianBulla** [reply]
Hello JWilliams,
this is great !
Thank you very much for letting it know. I'm sure it will help also others, who face similar issue.

Thanks you very much for it and enjoy your simulation results from OpenRadioss :-)

Thanks and best regards,
Marian
